.github/workflows/bump-elastic-stack-snapshot.yml

```
@@ -34,5 +37,7 @@ jobs:
           vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
           vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
           pipeline: ./.ci/bump-elastic-stack-snapshot.yml
+          notifySlackChannel: ""
+          messageIfFailure: ":traffic_cone: updatecli failed for `${{ github.repository }}@${{ github.ref_name }}`, @TBC please look what's going on <${{ env.JOB_URL }}|here>"
```

What slack channel and what team should be notified in the message.

```suggestion
          notifySlackChannel: '#my-new-channel'
          messageIfFailure: ":traffic_cone: updatecli failed for `${{ github.repository }}@${{ github.ref_name }}`, @my-new-team please look what's going on <${{ env.JOB_URL }}|here>"
```

---

`#ingest-notifications` is probably best

---

```suggestion
          notifySlackChannel: "#ingest-notifications"
          messageIfFailure: ":traffic_cone: updatecli failed for `${{ github.repository }}@${{ github.ref_name }}`, `@update-me-with-the-slack-team-to-be-poked` please look what's going on <${{ env.JOB_URL }}|here>"
```